src/Components/NewForm.vue

```diff
@@ -96,6 +71,32 @@
 					:checked.sync="allowComments">
 					{{ t('announcementcenter', 'Allow comments') }}
 				</NcActionCheckbox>
+				<NcActionSeparator />
+				<NcActionCheckbox value="0"
+					:checked.sync="scheduleEnabled">
+					{{ t('announcementcenter', 'Schedule announcement time (optional)') }}
+				</NcActionCheckbox>
+				<NcDateTimePicker v-model="scheduleTime"
```

Comment:
Handling of this feels quite complicated and e.g. there is no "easy" way to close it after you selected minutes.
Maybe we can do something closer to the reminder in Talk?
https://github.com/nextcloud/spreed/blob/main/src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue#L192-L200

Reply:
@nickvergessen I started implementing this, while this works as well I ran into a few issues:
- you need to select the hours or this component doesn't default to 0:00, which feels un-intuitive
- I can't get rid of that "Please choose a date" text, I updated the screenshot in the description